core/src/stored/CMakeLists.txt

```diff
@@ -219,8 +219,10 @@ target_link_libraries(stored_objects ${NDMP_LIBS})
 
 add_executable(bareos-sd stored.cc)
 
-set(BAREOS_SD_LIBRARIES bareos bareossd bareosfind ${NDMP_LIBS})
-
+set(BAREOS_SD_LIBRARIES
+    bareos bareossd bareosfind ${NDMP_LIBS} ${PTHREAD_LIBRARIES}
+    $<$<PLATFORM_ID:FreeBSD>:${JANSSON_LIBRARIES}>
```

Comment:
this look fishy, either jansson is used or it is not. I don't think it can be used only on FreeBSD